src/coffee/mapping.coffee

```diff
@@ -348,6 +350,18 @@ class Mapping
 
     prices
 
+  mapTiers: (tiers) ->
+    unless tiers
+      return []
+    tiers.split(GLOBALS.DELIM_TIERS_MULTI_VALUE).map((priceTier) ->
+      matchedPriceTier = priceTier.split(/ |@/g)
+      formattedTier =
+        value:
+          currencyCode: matchedPriceTier[0]
+          centAmount: parseInt matchedPriceTier[1]
```

Comment:
```suggestion
          centAmount: parseInt matchedPriceTier[1], 10
```
it is safer to use `parseInt(number, 10)` so it is always parsed with decimal number base